idem_azurerm/exec/azurerm/storage/container.py

```diff
@@ -475,6 +659,50 @@ async def list_(hub, ctx, account, resource_group, **kwargs):
     return result
 
 
+async def list_blobs(hub, ctx, container, account, resource_group, **kwargs):
+    """
+    .. versionadded:: 3.0.0
+
+    Returns a generator to list the blobs under the specified container. The generator will lazily follow the
```

Comment:
this is misleading. the SDK probably returns a generator, but you're returning a dict.